torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -59,6 +59,12 @@
 //   https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/templates/Operators.h
 //   (You can find some examples below)
 
+#if defined(USE_TORCH_NEW_OPTIONAL_GENERATOR_TYPE)
```

Comment:
Any reasons why we use a marco here?

Reply:
I don't think we need this guard. You can add a .torch_pin to this PR which will then pick up your PR on the upstream. So we can land those together!
https://github.com/pytorch/xla/tree/db64a45469acb20d6b75299b6434f9e11c1c95c2/torch_patches

Reply:
Fixed